go.mod

```diff
@@ -33,3 +33,5 @@ require (
 replace github.com/metal3-io/cluster-api-provider-metal3/api => ./api
 
 replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.4.4
+
+require github.com/opencontainers/image-spec v1.0.2 // indirect
```

Comment:
qq: Why not to add it in the **require** list?

Reply:
Wanted to make it separate so that its easy to remove in future